schedule/functional/extra_tests_gnome.yaml

```
@@ -0,0 +1,17 @@
+name:   extra_tests_gnome
+description:    >
+    Maintainer: asmorodskyi, okurz.
+    Extra tests which were designed to run on gnome
```

To avoid confusion, it should mention explicitly that is to test packages from Desktop applications module

```suggestion
    Extra tests for software in desktop applications module which were designed to run on gnome
```

---

sorry, forgot about that